src/app/app.component.html

```diff
@@ -1,114 +1,124 @@
+          <a routerLink="/stepper" (click)="toggleMobileSideMenu()">
+            <tds-icon name="report" size="24"></tds-icon>
+            Stepper
+          </a>
+        </tds-side-menu-item>
+        <tds-side-menu-item [attr.selected]="isActive('/table')">
+          <a routerLink="/table" (click)="toggleMobileSideMenu()">
+            <tds-icon name="document_tool" size="24"></tds-icon>
+            Table
+          </a>
+        </tds-side-menu-item>
+        <tds-side-menu-item class="mobile-nav-item" [attr.selected]="isActive('/components')">
+          <a routerLink="/components" (click)="toggleMobileSideMenu()">
+            <tds-icon name="tool" size="24"></tds-icon>
+            Components
+          </a>
+        </tds-side-menu-item>
+        <tds-side-menu-item class="mobile-nav-item" [attr.selected]="isActive('/form')">
+          <a routerLink="/form" (click)="toggleMobileSideMenu()">
+            <tds-icon name="edit" size="24"></tds-icon>
+            Form
+          </a>
+        </tds-side-menu-item>
+    
+        <tds-side-menu-dropdown class="mobile-nav-item" [defaultOpen]="isActive('/tabs')" [selected]="isActive('/tabs')">
+          <tds-icon slot="button-icon" name="folder" size="24"></tds-icon>
+          <span slot="button-label">Tabs</span>
+          <tds-side-menu-dropdown-list>
+            <tds-side-menu-dropdown-list-item [selected]="isActive('/tabs-buttons')">
+              <a (click)="toggleMobileSideMenu()" routerLink="/tabs-buttons">Button</a>
+            </tds-side-menu-dropdown-list-item>
+            <tds-side-menu-dropdown-list-item [selected]="isActive('/tabs-link')">
+              <a (click)="toggleMobileSideMenu()" routerLink="/tabs-links">Link</a>
+            </tds-side-menu-dropdown-list-item>
+          </tds-side-menu-dropdown-list>
+        </tds-side-menu-dropdown>
+    
+    
+    
+        <tds-side-menu-item  class="mobile-nav-item" [attr.selected]="isActive('/text')">
+          <a routerLink="/text" (click)="toggleMobileSideMenu()">
+            <tds-icon name="document" size="24"></tds-icon>
+            Text
+          </a>
+        </tds-side-menu-item>
+    
+    
+        <tds-side-menu-item slot="end">
+          <button>
+            <tds-side-menu-user heading="Name Namesson" subheading="Company name">
+              <img slot="image" src="https://www.svgrepo.com/show/384676/account-avatar-profile-user-6.svg"
+                alt="User menu." />
+            </tds-side-menu-user>
+          </button>
+        </tds-side-menu-item>
+    
+        <tds-side-menu-collapse-button slot="sticky-end"></tds-side-menu-collapse-button>
+      </tds-side-menu>
+      <main class="tds-u-h-100 tds-u-w-100">
+        <div class="tds-u-h-100 tds-u-p3">
+          <app-mode-switcher (modeToggle)="handleModeToggle()"></app-mode-switcher>
+          <app-mode-variant-switcher (modeVariantToggle)="handleModeVariantToggle()"></app-mode-variant-switcher>
```

Comment:
```suggestion
<span class="tds-u-flex tds-u-flex-gap-1 tds-u-flex-dir-col tds-u-pb3">
          <app-mode-switcher (modeToggle)="handleModeToggle()"></app-mode-switcher>
          <app-mode-variant-switcher (modeVariantToggle)="handleModeVariantToggle()"></app-mode-variant-switcher>
          </span>
```

Just to make some spacing between them and after them...